src/function.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable-next-line import/no-cycle */
```

Comment:
- - You have done great so far. adding, removing, and editing tasks works great. :+1: :100:  But there are some changes you need to make before moving to the next milestone. :smile: 
- [ ]  As required by Microverse, you are not meant to use any linters disable rule. The linters are setup to make sure you follow the best industry standard practices. Therefore I'll suggest you remove the disable rule and fix the issue. :smile: :wave: